tests/php/Analyzer/TupleSymbol/QuoteSymbolTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phel\Analyzer\TupleSymbol;
+
+use Phel\Exceptions\PhelCodeException;
+use Phel\Lang\Symbol;
+use Phel\Lang\Tuple;
+use Phel\NodeEnvironment;
+use PHPUnit\Framework\TestCase;
+
+final class QuoteSymbolTest extends TestCase
+{
+    /** @test */
```

Comment:
Changed the annotation by the method name prefix.

I added the guard-clause condition in `QuoteTuple` class, in order to ensure the current object is, in fact, a `QuoteSymbol` (as it was already done in `LoopSymbol`).

From my point of view, the `if` conditional has a micro computational cost but the class is much more secure.
I think it worths to avoid possible conflicts in the future. And with PHP 8 and JIT feature will be much faster!

I add a one more test to check that.